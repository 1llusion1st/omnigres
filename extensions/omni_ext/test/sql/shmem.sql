-- Since it's preloaded, it's already initialized
SELECT omni_ext_test.alloc_shmem_global();

--- Database-local
SELECT omni_ext_test.alloc_shmem_database_local();

CREATE DATABASE another_db;
\connect another_db
CREATE EXTENSION omni_ext_test CASCADE;
SELECT omni_ext_test.alloc_shmem_database_local();