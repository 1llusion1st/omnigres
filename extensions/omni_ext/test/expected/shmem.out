-- Since it's preloaded, it's already initialized
SELECT omni_ext_test.alloc_shmem_global();
 alloc_shmem_global 
--------------------
 updated
(1 row)

--- Database-local
SELECT omni_ext_test.alloc_shmem_database_local();
 alloc_shmem_database_local 
----------------------------
 testdb 0
(1 row)

CREATE DATABASE another_db;
\connect another_db
CREATE EXTENSION omni_ext_test CASCADE;
NOTICE:  installing required extension "omni_ext"
SELECT omni_ext_test.alloc_shmem_database_local();
 alloc_shmem_database_local 
----------------------------
 testdb 1
(1 row)

